lib/mongo/auth/stringprep.rb

```diff
@@ -0,0 +1,66 @@
+require 'mongo/auth/stringprep/tables'
+
+module Mongo
+  module Auth
+    module StringPrep
+      extend self
+
+      def prepare(data, mappings, prohibited, options = {})
+        out = apply_maps(data, mappings)
```

Comment:
is out the "mapped" data? If so, can we called it "mapped"?

Reply:
Done